compiler/quilt/tools/command.py

```diff
@@ -488,7 +493,7 @@ def _log(team, **kwargs):
     # restore disabled error-handling
     session.hooks['response'] = orig_response_hooks
 
-def build(package, path=None, dry_run=False, env='default'):
+def build(package, path=None, dry_run=False, env='default', force=False):
```

Comment:
You should probably add a `--force` to main, too - otherwise, you can't set it from the CLI?

Reply:
I think CLI should not need force, if the prompt becomes such a pain point then they should log out and log in with their public cloud account. We want to avoid people just jamming --force on everything to not have to deal with prompts.

Reply:
I can add it but I left it off intentionally, if you disagree I'll add it.

Reply:
ah, that's fine then